apps/services/auth/api/src/app/v1/delegations/delegations.controller.spec.ts

```diff
@@ -341,6 +382,18 @@ describe('DelegationsController', () => {
                 )
               })
             })
+
+            it('should return delegation if national registry api getIndividual throws and error', async () => {
```

Comment:
What is this testing?

Reply:
This is making sure that delegations get return although the national registry throws an error.